teamsupport/utils.py

```diff
@@ -7,6 +7,6 @@ def to_xml(root, data):
 
     for k, v in iteritems(data):
         child = etree.SubElement(xml, k)
-        child.text = v
+        child.text = str(v)
```

Comment:
> `text`: Text before the first subelement. This is either a string or the value None, if there was no text.

from http://lxml.de/api/lxml.etree._Element-class.html

This should probably check for `None` before converting to a string.

Reply:
and what if it `is None`?

Reply:
`text = ''`?

Reply:
I would expect that `child.text = None` and `child.text = ''` would generate equivalent XML.  Try it and find out.

Reply:
This is python 2, shouldn't this be unicode?

Reply:
Its 2 and 3 actually so `six.text_type`.